vagrant/README.md

```diff
@@ -116,26 +171,109 @@ default   backend    service=backend version=v0   Online   6m3s
+
+```yaml
+type: Dataplane
+mesh: default
+name: kong
+networking:
+  gateway:
+    tags:
+      service: kong
+  outbound:
+  - interface: :28080
+    service: frontend 
+```
+The inbound networking configuration is replaced with gateway to enable Gateway mode. And the outbound points to our [frontend service](/README.md#frontend), similar to how we configured Kong in the [previous section](#configure-and-start-kong). 
+
+With this dataplane policy, [`register_dataplane.sh`](/vagrant/common/register_dataplane.sh) will apply this configuration to the dataplane that will sit alongside our Kong gateway. 
+
+### Prometheus
+
+Out-of-the-box, Kuma provides full integration with Prometheus:
+* if enabled, every dataplane will expose its metrics in Prometheus format
+* furthemore, Kuma will make sure that Prometheus can automatically find every dataplane in the mesh
+
+In this demo guide, Prometheus is already deployed and configured. All the files that are used to configure and deploy Prometheus can be found in the [`/vagrant/prometheus/`](/vagrant/prometheus/app/install.sh) directory. If you need more help, refer to the official documentation [here](https://kuma.io/docs/0.3.2/policies/#traffic-metrics). **To enable Prometheus metrics on every dataplane in the mesh, configure a mesh resource as shown in the [Traffic Metric Policy](#traffic-metrics) section.**
+
+Once [Traffic Metric policies](#traffic-metrics) are added, you can visit the [Prometheus dashboard](http://192.168.33.80:9090/) to query the metrics that Prometheus is scraping from our Kuma mesh.
+
+## Policies
+
+### mTLS
+
+This policy enables automatic encrypted mTLS traffic for all the services in a mesh. Kuma ships with a builtin CA (Certificate Authority) which is initialized with an auto-generated root certificate. The root certificate is unique for every mesh and it used to sign identity certificates for every data-plane. Kuma also supports third-party CA.
+
+By default, mTLS is not enabled. You can enable Mutual TLS by updating the mesh policy with the mtls setting.
+
+#### Check for mTLS
+
+Using [`kumactl`](#kumactl) that you configured earlier, you can check the mesh resource and see that mTLS is turned off. You can also visualize these resource by inspecting them in the [GUI](#gui).
+
+```bash
 $ ./kumactl get meshes
 NAME      mTLS   CA        METRICS
 default   off    builtin   off
 ```
 
-### 9. View our application:
+#### Adding mTLS Policy
 
-To shop at Kuma's marketplace, access the Kong gateway that is the ingress to your mesh at [http://192.168.33.70:8000](http://192.168.33.70:8000). All the traffic between the machines are routed through Kuma's dataplane.
+Use `kumactl apply [..]` to enable mTLS on our mesh.
 
-### 10. Let's enable mTLS using `kumactl`:
-
-```
+```bash
 $ cat <<EOF | kumactl apply -f -
-type: Mesh
+type: mesh
```

Comment:
Let's revert this change:
* `type: mesh` is NOT valid because `Mesh` must begin with a capital `M`

Reply:
ahh i did a mass replace so these got changed, fixed